src/core/client/stream/tabs/Configure/OpenOrCloseStream/OpenOrCloseStreamContainer.tsx

```diff
@@ -48,6 +48,10 @@ const OpenOrCloseStreamContainer: FunctionComponent<Props> = ({
     setShowSuccess(false);
   }, [setShowSuccess]);
 
+  if (story.isArchiving || story.isArchived) {
+    return null;
+  }
```

Comment:
This check seems now to be obsolete as this Component is longer rendered when story isArchived or in progress of archiving. If you wanna a check here, I'd recommend you to render an error instead.